### PR TITLE
Closes #498: Fields with parent translations

### LINKED ISSUE #498: Fix problems with concatenating parent and child metadata field names

### PR DESCRIPTION
**Pull request by dbojanek**

### CODE REVIEW

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dashboard/DashboardUsersService.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    public DashboardUsersService() {
+    }
+
+    @Inject
+    public DashboardUsersService(EjbDataverseEngine commandEngine, DataverseRequestServiceBean dvRequestService) {
+        this.commandEngine = commandEngine;
+        this.dvRequestService = dvRequestService;
+    }
+
+    // -------------------- LOGIC --------------------
```
Comment (MrK191) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dashboard/DashboardUsersService.java]: No logic in this class, is it needed?

File: dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetFieldType.java [hunk 2]
```diff
@@ -520,7 +522,7 @@ public int compareTo(DatasetFieldType o) {
 
     public String getDisplayName() {
         if (isHasParent() && !parentDatasetFieldType.getTitle().equals(title)) {
-            return parentDatasetFieldType.getLocaleTitle() + " " + getLocaleTitle();
+            return Optional.ofNullable(getLocaleTitleWithParent()).filter(s -> !s.isEmpty()).orElse(parentDatasetFieldType.getLocaleTitle() + " " + getLocaleTitle());
```
Comment (MrK191) [inline dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetFieldType.java:525]: Was there a bug before with this?

**Review by MrK191 — CHANGES_REQUESTED** [review]

File: dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetFieldType.java (hunk 2 above)
Comment (dbojanek) [reply dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetFieldType.java:525]: There was no bug, but now logic changes. If there is a translation for field with parent, then we get that. Else we use old logic of concatenating parent + child values (for example for now @wfenrich didn't make english translations for fields with parents)

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dashboard/DashboardUsersService.java (hunk 1 above)
Comment (dbojanek) [reply dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dashboard/DashboardUsersService.java]: This was changed in another task. I've merged with develop so it's up to date now.

**Review by MrK191 — APPROVED** [review]

**Pull request merged**